prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```
@@ -155,29 +155,29 @@ elements:
     hl7Spec: [ '%{hl7OBXField}-19' ]
 
   - name: performing-organization-name-pracrole
-    condition: '%observation.performer.resolve() is PractitionerRole'
-    resource: '%resource.performer.resolve().organization.resolve()'
+    condition: '%observation.performer.resolve().ofType(PractitionerRole).exists()'
+    resource: '%resource.performer.resolve().ofType(PractitionerRole).organization.resolve()'
     schema: ../../common/datatype/xon-organization
     constants:
       hl7OrgField: '%{hl7OBXField}-23'
 
   - name: performing-organization-name-non-pracrole
-    condition: '%observation.performer.resolve().is(PractitionerRole).not()'
+    condition: '%observation.performer.resolve().ofType(PractitionerRole).exists().not()'
     resource: '%resource.performer.resolve()'
     schema: ../../common/datatype/xon-organization
     constants:
       hl7OrgField: '%{hl7OBXField}-23'
 
   - name: performing-organization-address-pracrole
-    condition: '%observation.performer.resolve() is PractitionerRole'
-    resource: '%resource.performer.resolve().organization.resolve().address'
+    condition: '%observation.performer.resolve().ofType(PractitionerRole).exists()'
+    resource: '%resource.performer.resolve().ofType(PractitionerRole).organization.resolve().address'
     constants:
       hl7AddressField: '%{hl7OBXField}-24'
     schema: ../../common/datatype/xad-extended-address
     resourceIndex: contactIndex
 
   - name: performing-organization-address-non-pracrole
-    condition: '%observation.performer.resolve().is(PractitionerRole).not()'
+    condition: '%observation.performer.resolve().ofType(PractitionerRole).exists().not()'
```

Similar issue to above looks like it would exist here, but it's less straightforward to resolve. Maybe the existence of a performer.address is enough to tell that the performer is an Organization, so this would work?
```suggestion
    condition: '%resource.exists and %observation.performer.resolve().ofType(Organization).exists()'
```